Gemfile

```diff
@@ -8,6 +8,9 @@ gem 'rspec'
 gem 'coveralls', require: false
 
 gem 'psych', platforms: :rbx
+gem 'rack', '~> 1.6.5'
+gem 'tins', '~> 1.6.0'
+gem 'term-ansicolor', '~> 1.3.2'
 
```

Comment:
Ah okay, I guess they might not be needed since updated the Ruby versions on CI.